src/main/java/com/revature/controllers/PersonController.java

```diff
@@ -84,9 +84,15 @@
 	
 	@RequestMapping(method = RequestMethod.POST, value = "persons")
 	public ResponseEntity<Person> createPerson(@RequestBody Person newPerson){
+		if (newPerson.getClass() != Person.class){
+			return new ResponseEntity<>(HttpStatus.BAD_REQUEST);
+		} else{
+			
 		System.out.println(newPerson);	
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS106)